lib/features/recommendation/interests/widgets/interest_card.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:givt_app_kids_web/features/recommendation/tags/models/tag.dart';
+import 'package:givt_app_kids_web/utils/font_utils.dart';
+
+class InterestCard extends StatelessWidget {
+  const InterestCard({
+    required this.interest,
+    required this.width,
+    required this.onPressed,
+    this.isSelected = false,
+    super.key,
+  });
+
+  final Tag interest;
+  final void Function() onPressed;
+  final double width;
+  final bool isSelected;
+
+  @override
+  Widget build(BuildContext context) {
+    final size = MediaQuery.sizeOf(context);
+    return Container(
+      padding: EdgeInsets.all(width * 0.02),
+      child: ElevatedButton(
+        style: ElevatedButton.styleFrom(
+          elevation: 25,
+          shape: RoundedRectangleBorder(
+            borderRadius: BorderRadius.circular(size.width * 0.020),
+            side: isSelected
+                ? const BorderSide(
+                    color: Color(0xFF97A486),
```

Comment:
Should we start putting the colors in one file? To make it easier to change thi gs

Reply:
I agree) In regard of speed up testing process can I suggest to make a separate task about moving all colors in the app in a theme file?